config/application.yml.default

```diff
@@ -132,7 +132,6 @@ idv_send_link_attempt_window_in_minutes: 10
 idv_send_link_max_attempts: 5
 idv_tmx_test_csp_disabled_emails: '[]'
 idv_tmx_test_js_disabled_emails: '[]'
-ie11_support_end_date: '2022-12-31'
```

Comment:
Need to remove the IdentityConfig reference as well:

https://github.com/18F/identity-idp/blob/714f4c76891a27582a4456e883aca3807ff87c5f/lib/identity_config.rb#L227